lib/identity_config.rb

```diff
@@ -177,9 +177,17 @@ def self.build_store(config_map)
     config.add(:use_dashboard_service_providers, type: :boolean)
     config.add(:use_kms, type: :boolean)
     config.add(:usps_confirmation_max_days, type: :integer)
+    config.add(:usps_ipp_password, type: :string)
+    config.add(:usps_ipp_root_url, type: :string)
+    config.add(:usps_ipp_sponsor_id, type: :string)
+    config.add(:usps_ipp_username, type: :string)
     config.add(:usps_upload_enabled, type: :boolean)
-    config.add(:valid_authn_contexts, type: :json)
-    final_env = config.add(:usps_ipp_root_url)
+    config.add(:usps_upload_sftp_directory, type: :string)
+    config.add(:usps_upload_sftp_host, type: :string)
+    config.add(:usps_upload_sftp_password, type: :string)
+    config.add(:usps_upload_sftp_timeout, type: :integer)
+    config.add(:usps_upload_sftp_username, type: :string)
+    final_env = config.add(:valid_authn_contexts, type: :json)
 
     @store = RedactedStruct.new('IdentityConfig', *final_env.keys, keyword_init: true).
```

Comment:
we could reduce diff noise in this section of the file if we used @written_env

```suggestion
    config.add(:valid_authn_contexts, type: :json)

    @store = RedactedStruct.new('IdentityConfig', *@written_env.keys, keyword_init: true).
```

Reply:
That doesn't _exactly_ work because this is not an instance method. We'd have to expose `@written_env` to callers. I'll push a commit where I add an `attr_reader` and we can see how we like it.